packages/core/src/com/hosts/ws-client-host.ts

```diff
@@ -11,10 +11,13 @@ export class WsClientHost extends BaseHost {
     constructor(url: string, options?: Partial<SocketOptions>) {
         super();
 
+        const { protocol, host, pathname: path, searchParams } = new URL(url);
+        const query = Object.fromEntries(searchParams as unknown as Iterable<any>);
```

Comment:
Object.fromEntries(searchParams.entries())

Reply:
I tried but for some reason typescript doesn't know that entries are there

![image](https://user-images.githubusercontent.com/91947555/179922689-b27c9bc7-c804-47e9-bd9c-577db304339a.png)